test/development/acceptance-app/ReactRefreshLogBoxMisc.test.ts

```diff
@@ -3,210 +3,15 @@ import { FileRef, nextTestSetup } from 'e2e-utils'
 import path from 'path'
 import { outdent } from 'outdent'
 
-// TODO: re-enable these tests after figuring out what is causing
-// them to be so unreliable in CI
-describe.skip('ReactRefreshLogBox app', () => {
-  const { next } = nextTestSetup({
+describe('ReactRefreshLogBox app', () => {
+  const { isTurbopack, next } = nextTestSetup({
     files: new FileRef(path.join(__dirname, 'fixtures', 'default-template')),
     skipStart: true,
   })
 
-  test('<Link> with multiple children', async () => {
-    await using sandbox = await createSandbox(next)
-    const { session } = sandbox
-
-    await session.patch(
-      'index.js',
-      outdent`
-        import Link from 'next/link'
-
-        export default function Index() {
-          return (
-            <Link href="/">
-              <p>One</p>
-              <p>Two</p>
-            </Link>
-          )
-        }
-      `
-    )
-
-    await session.assertHasRedbox()
-    expect(await session.getRedboxDescription()).toMatchInlineSnapshot(
-      `"Error: Multiple children were passed to <Link> with \`href\` of \`/\` but only one child is supported https://nextjs.org/docs/messages/link-multiple-children"`
-    )
-    expect(
-      await session.evaluate(
-        () =>
-          (
-            document
-              .querySelector('body > nextjs-portal')
-              .shadowRoot.querySelector(
-                '#nextjs__container_errors_desc a:nth-of-type(1)'
-              ) as any
-          ).href
-      )
-    ).toMatch('https://nextjs.org/docs/messages/link-multiple-children')
-  })
-
-  test('<Link> component props errors', async () => {
```

Comment:
Tested in https://github.com/vercel/next.js/blob/aae3692ce7b473716690a71cbd54c7d7e3552abd/test/e2e/next-link-errors/next-link-errors.test.ts#L13